common/scripts/kind_provisioner.sh

```diff
@@ -183,13 +183,22 @@ EOF
   fi
 
   # Create KinD cluster
-  if ! (kind create cluster --name="${NAME}" --config "${CONFIG}" -v4 --retain --image "${IMAGE}" --wait=180s); then
+  if ! (kind create cluster --name="${NAME}" --config "${CONFIG}" -v4 --retain --image "${IMAGE}" ); then
     echo "Could not setup KinD environment. Something wrong with KinD setup. Exporting logs."
     return 9
   fi
   # Workaround kind issue causing taints to not be removed in 1.24
   kubectl taint nodes "${NAME}"-control-plane node-role.kubernetes.io/control-plane- || true
 
+  # If using the ambient config we'll need to install our own CNI
+  # TODO: Will need updating when mc ambient tests are created if we want our own CNI
+  if [[ "$CONFIG" = *"ambient-sc.yaml" ]]; then
```

Comment:
We should not hardcode like this, make calico explicit opt in

Reply:
We could do something like if CNI=Calico then patch the config, spin a cluster, install Calico